content/docs/tasks/security/authn-policy/index.md

```diff
@@ -448,19 +489,17 @@ $ curl $INGRESS_HOST/headers -s -o /dev/null -w "%{http_code}\n"
 200
 {{< /text >}}
 
-Now, let's add a policy that requires end-user JWT for `httpbin.foo`. The next command assumes policy with name "httpbin" already exists (which should be if you follow previous sections). You can run `kubectl get policies.authentication.istio.io -n foo` to confirm, and use `istio create` (instead of `istio replace`) if resource is not found. Also note in this policy, peer authentication (mutual TLS) is also set, though it can be removed without affecting origin authentication settings.
+Now, let's add a policy that requires end-user JWT for `httpbin.foo`. The next command assumes there is no service-specific policy for `httpbin.foo` (which should be if you run [cleanup](#cleanup-part-2) as instructed). You can run `kubectl get policies.authentication.istio.io -n foo` to confirm.
```

Comment:
Now, add a policy
which should be -> which should be the case
instructed -> described